RELEASE_NOTES.md

```diff
@@ -2,8 +2,7 @@ Release notes
 =============
 
 ### Next
-* Fixes an issue with trust anchors for eIDAS metadata resolvers not being updated when they changed at source. The VSP
-  will now update them without requiring a restart.
+* Removed processing of assertions received from EU Member States. The configuration parameter `europeanIdentity`, if present, and child elements must be removed in order for the VSP to start.
```

Comment:
I think I'm confused that we're still imposing this backwards-incompatible change on services - have they all confirmed this is ok? If not can't we split that off?

Reply:
Unlike the MSA, we can remove all config from the VSP, and it won't affect the processing of Verify SAML assertions. The presence of eIDAS config in the MSA changes the payload sent to the local matching service.

I think it's reasonable to tell VSP users they have to remove all eIDAS config along with the upgrade.

Reply:
Aha, I was confused which PR I was reading :-D

Reply:
(To be honest I don't really see why we can't just _ignore_ the `europeanIdentity` config rather than insist they make a change, but :shrug: :-) )